api/Hmcr.Data/Repositories/ActivityCodeRepository.cs

```diff
@@ -29,5 +37,92 @@ public async Task<IEnumerable<ActivityCodeDto>> GetActiveActivityCodesAsync()
 
             return Mapper.Map<IEnumerable<ActivityCodeDto>>(activities).Where(x => x.IsActive);
         }
+
+        public async Task<HmrActivityCode> CreateActivityCodeAsync(ActivityCodeCreateDto activityCode)
+        {
+            var activityCodeEntity = new HmrActivityCode
```

Comment:
Please use AutoMap instead of assigining values manually.

Reply:
Of course. I looked at CreateUserAsync as an example and in there it didn't use the Mapper.